soh/include/z64.h

```diff
@@ -1215,18 +1215,45 @@ typedef struct {
 struct SelectContext;
 
 typedef struct {
-    /* 0x00 */ char* name;
+    /* 0x00 */ char* japaneseName;
+    /*      */ char* englishName;
+    /*      */ char* germanName;
+    /*      */ char* frenchName;
     /* 0x04 */ void (*loadFunc)(struct SelectContext*, s32);
     /* 0x08 */ s32 entranceIndex;
 } SceneSelectEntry; // size = 0xC
```

Comment:
sizes here are wrong now that we've got 4 chars in here, probably worth it to remove the sizes or update them to be accurate